tools/update_assisted_installer_yaml.py

```diff
@@ -11,22 +12,56 @@
 args = parser.parse_args()
 
 
+def in_all_lists(el: str, lists: list[str]) -> bool:
+    for list in lists:
+        if el not in list:
+            return False
+    return True
+
+
+def find_first_common_element(lists: list[str]) -> str:
+    if len(lists) < 1:
+        raise IndexError("No lists provided. Cannot find common element")
+    for el in lists[0]:
+        if in_all_lists(el, lists[1:]):
+            return el
+    raise RuntimeError("Could not find common element")
+
+
+def get_ref_by_docker_image(images):
+    tag_pattern = "^latest-[a-f0-9]{40}$"
+    tag_prefix = "latest-"
+    sk = skopeo.Skopeo()
+    image_tags = []
+    for img in images:
+        image_tags.append(sk.get_image_tags_by_pattern(img, tag_pattern))
+    if len(image_tags) < 1:
```

Comment:
I think it's more explicit to do:
```suggestion
    if len(image_tags) == 0:
```